app/js/components/subAddress/addressForm.jsx

```diff
@@ -169,20 +150,40 @@ export default class AddressForm extends React.Component {
     return (
       <div>
         <div>
-          {
-            this.state.isShowingModal &&
-            <ModalContainer onClose={this.handleClose}>
-              <ModalDialog onClose={this.handleClose}>
-                <h1>Dialog Address</h1>
-                <p>Are you sure you want to delete? <br />
-                  Click yes to delete or close this modal to cancel</p>
-                <div>
-                  <button type="button" name="modalDelete" onClick={this.delete}
-                  className="btn btn-default form-control cancelBtn">Yes</button>
+          <div className="modal fade" id="myModal" role="dialog">
+            <div className="modal-dialog">
+              <div className="modal-content">
+                <div className="modal-header">
+                  <button type="button" className="close" data-dismiss="modal">&times;</button>
+                  <h4 className="modal-title">Confirm Delete</h4>
                 </div>
-              </ModalDialog>
-            </ModalContainer>
-          }
+                <div className="modal-body">
+                  <p> Are you sure want to delete this address?</p>
+                </div>
```

Comment:
Seems theres a typo on the prompt message. "Are you sure **you** want to delete this address?